### PR TITLE
chore: adding approved icons to buttons and styling passwords

### PR DESCRIPTION
**Pull request by rasoul-shahriari**

### CODE REVIEW

File: src/modules/profile/infos-main.tsx [hunk 1]
```diff
@@ -79,7 +85,12 @@ export const InfosMain = observer(() => {
           size="big"
           onPress={() => navigation.navigate('profile-infos-phone')}
         >
-          {profileFlowStore.phone && profileFlowStore.dialCode ? '✓' : ''} Phone
+          {profileFlowStore.phone && profileFlowStore.dialCode ? (
+            <ApprovedIcon />
+          ) : (
+            ''
+          )}{' '}
```
Comment (mfateminia) [inline src/modules/profile/infos-main.tsx]: same?

File: src/modules/profile/infos-main.tsx [hunk 2]
```diff
@@ -90,7 +101,7 @@ export const InfosMain = observer(() => {
             theme="gray"
             onPress={() => navigation.navigate('profile-infos-address')}
           >
-            {profileFlowStore.address ? '✓' : ''} Address
+            {profileFlowStore.address ? <ApprovedIcon /> : ''} Address
```
Comment (mfateminia) [inline src/modules/profile/infos-main.tsx]: same?

File: src/modules/auth/passcode/password.tsx [hunk 3]
```diff
@@ -27,7 +38,11 @@ interface DotProps {
 }
 
 const Dot: FC<DotProps> = memo(({ active }) => {
-  return <View style={[styles.dot, active && styles.dotActive]} />;
+  return (
+    <View style={[styles.dot, active && styles.dotActive]}>
+      {active && <View style={[styles.circle]} />}
```
Comment (Mokto) [inline src/modules/auth/passcode/password.tsx]: `style={styles.circle}`

File: src/router.tsx [hunk 4]
```diff
@@ -21,9 +21,9 @@ export const Router = memo(() => {
         cardOverlayEnabled: false,
       }}
     >
-      <Stack.Screen name="intro" component={Intro} />
+      {/* <Stack.Screen name="intro" component={Intro} />
```
Comment (Mokto) [inline src/router.tsx]: cancel that please

File: src/modules/profile/infos-main.tsx [hunk 5]
```diff
@@ -63,14 +65,18 @@ export const InfosMain = observer(() => {
           size="big"
           onPress={() => navigation.navigate('profile-infos-name')}
         >
-          {profileFlowStore.firstName && profileFlowStore.lastName ? '✓' : ''}{' '}
+          {profileFlowStore.firstName && profileFlowStore.lastName ? (
```
Comment (mfateminia) [inline src/modules/profile/infos-main.tsx]: how about ```profileFlowStore.firstName && profileFlowStore.lastName && <ApprovedIcon />```

**Pull request merged**